src/style.css

```
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo {
+  padding-top: 3px;
+}
+
+.nav-menu {
+  display: flex;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav-list {
+  display: block;
+  list-style: none;
+}
+
+.nav-link {
+  color: #000;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 14.06px;
+  text-decoration: none;
+  font-weight: medium;
+  position: relative;
+  display: flex;
+  height: 60px;
+  margin-left: 20px;
+  align-items: center;
+}
+
+.is-active {
+  color: #00acdc;
+  margin-left: 0;
```

```suggestion
  margin-left: 0;
```
let's use `:first-child` selector for resetting margin-left